rmgpy/molecule/group.py

```diff
@@ -1874,6 +1955,24 @@ def is_subgraph_isomorphic(self, other, initial_map=None, generate_initial_map=F
                         return False
         else:
             if group.multiplicity: return False
+        if self.metal:
+            for mult1 in self.metal:
```

Comment:
I defined m1,m2 as strings and use those instead of mult1,mult2 (that are shorts) for metal and facet. Sharing them for metal and facet marginally reduces the amount of memory allocated...it probably doesn't matter, but I think the code is clear without additional variables.